strr-platform-web/app/pages/auth/account/choose-existing.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+// import { isoCountriesList } from '~/utils/isoCountriesList'
+const localePath = useLocalePath()
+const { t } = useI18n()
+const isSmallScreen = useMediaQuery('(max-width: 640px)')
+const strrModal = useStrrModals()
+const accountStore = useConnectAccountStore()
+
+useHead({
+  title: t('platform.title.chooseAccount')
+})
+
+definePageMeta({
+  middleware: ['auth']
+})
+
+// display premium accounts first
+const sortedAccounts = computed(() => {
+  return accountStore.userAccounts.sort((a, b) => {
+    if (a.accountType === AccountType.PREMIUM && b.accountType !== AccountType.PREMIUM) {
+      return -1
+    } else if (a.accountType !== AccountType.PREMIUM && b.accountType === AccountType.PREMIUM) {
+      return 1
+    }
+    return 0
+  })
+})
+// const sortedAccounts = []
+
+// TODO: how do we want to redirect if the user has/does not have any platforms?
+// check if the account has a platform here and either redirect to the dashboard or registration page?
+// or redirect to the dashboard page and if the user doesnt have any platforms the dashboard can handle the redirect ?
+function handleAccountSwitch (id: string) {
+  const route = useRoute()
+  accountStore.switchCurrentAccount(id)
+  // TODO: add Strr product to org product list
+  // only navigate if account successfully changed
+  // if (accountStore.currentAccount.id === id) { // TODO: assume account change is always successful ?
+
+  if (route.query.return) {
+    return navigateTo(route.query.return as string)
+  } else {
+    return navigateTo(localePath('/platform/dashboard'))
+  }
+}
+
+// TODO: how much do we care about displaying the account mailing address ?
```

Comment:
I don't think we need to display this, but we probably want to display the branch info? And I think the branch info should also be in the header dropdown, so worth adding to the layer

Reply:
No address - branch is required